pom.xml

```diff
@@ -102,7 +102,7 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins.workflow</groupId>
             <artifactId>workflow-cps</artifactId>
-            <version>2.41</version>
+            <version>2.48</version> <!-- JENKINS-50752 fix is required for writeMavenPom() -->
```

Comment:
Could be left as a comment in the PR—no one in the future is going to give a darn.